frontend/src/components/pagedItemsTable/PagedItemsTable.tsx

```diff
@@ -117,23 +118,109 @@ function PagedItemsTable<T extends object>({
 		[dispatch, setPageSizeCallback]
 	)
 
+	const shiftHeld = useRef(false)
```

Comment:
What about sortPriority [0, 1] instead of shiftHeld to use directly in the sortByList...